Resource.js

```diff
@@ -250,6 +253,7 @@ module.exports = function(app, route, modelName, model) {
             .exec(function(err, items) {
               if (err) return this.respond(res, 500, err);
               res.status(res.statusCode).json(items);
+              next()
```

Comment:
Please add a semi-colon after each ```next()``` statement. While it is valid JavaScript, it is asked for within the code standards that I follow for Node.js projects (http://nodeguide.com/style.html#semicolons).  It also would make jshint complain. Other than that, this looks good to me, and I appreciate your contribution!